Ambiguous-Squids/Bangazon_initial_site
# Completed User registration page

# Description
Added a form, template, view and url to register a new user

# Related Ticket(s)
#22 

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x] I certify that all existing tests pass

## Documentation

[ ] There is new documentation in this pull request that must be reviewed..

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

```sh
git pull --prune
git checkout <feature_branch>
```

## Steps to Test
Outline the steps to test the code in this pull request.

1. Attempt to register a new user at login page
2. Input user information
3. Check in admin to see if new user has registered
